01_getting-started/02_installation/01_local-docker.md

```diff
@@ -4,8 +4,30 @@ title: Local - Docker
 nav_order: 1
 grand_parent: Getting started
 parent: Installation
-last_modified_date: 20-10-2021 13:00
+last_modified_date: 4-11-2021 14:00
 permalink: /getting-started/installation/local-docker/
 ---
 
-# Local: Docker
\ No newline at end of file
+# Local: Docker
+
+Prerequisite: [Docker](https://www.docker.com/) must be installed.
+
+1. Clone the MasaCMS repository from [GitHub](https://github.com/MasaCMS/MasaCMS) to you local machine.
+
+2. Goto the directory in which you cloned the MasaCMs Repository with a commandline tool (like Bash).
+
+3. Goto the subfolder `\core\docker\`. In this folder you can find several Docker configurations for different database flavours.
+    * `local-mssql`  for Microsoft SQL Server
+    * `local-mysql`  for MySQL
+    * `local-oracle` for Oracle
+    * `local-postgresql` for PostgreSQL
+
+4. Goto the subfolder of the database you want to use.
+
+5. Start the Docker image by typing `docker-compose up`. A Docker image will now be created and started, with the database of you choice.
+
```

Comment:
Add a warning alert for Mac users that they will have to change the port number for the database container, since that port is already in use by the OS itself.